### PR TITLE
#151991941 Refactor implementation of czar draws card

### PR DESCRIPTION
**Pull request by tosmak16**
#### What does this PR do?
it refactors the implementation of the feature czars should be able to draw a black card.

#### Description of Task to be completed?
* Change modal background color to transparent.
* Remove setTimeout in drawCard method
* Correct in game user notification.
#### How should this be manually tested?
* Click the start button
* Click any of the black cards when the modal pops up
#### Any background context you want to provide?
none for now.
#### What are the relevant pivotal tracker stories?
#151991941

#### Screenshots
image

### CODE REVIEW

File: config/socket/game.js [hunk 1]
```diff
@@ -5,12 +5,12 @@ import * as questions from '../../app/controllers/questions';
 import * as answers from '../../app/controllers/answers';
 
 const guestNames = [
-  'Disco Potato',
-  'Silver Blister',
-  'Insulated Mustard',
-  'Funeral Flapjack',
-  'Toenail',
-  'Urgent Drip',
+  'Daddy yo',
```
Comment (temilaj) [inline config/socket/game.js:8]: Nice, Good humor!

File: public/js/controllers/game.js [hunk 2]
```diff
@@ -1,205 +1,213 @@
...
+                }).text(link);
+              }, 200);
+              $scope.modalShown = true;
+            }
           }
         }
+      });
+
+      if ($location.search().game && !(/^\d+$/).test($location.search().game)) {
+        console.log('joining custom game');
```
Comment (temilaj) [inline public/js/controllers/game.js]: Why are we still doing?

**Review by temilaj — CHANGES_REQUESTED** [review]
made some comments please review

File: public/js/controllers/game.js (hunk 2 above)
Comment (tosmak16) [reply public/js/controllers/game.js]: Hmmm...'ll remove it now.

**Pull request merged**